working-together/README.md

```diff
@@ -182,6 +182,15 @@ We've compiled some guidelines from various sources to help people feel included
   - Use the chat for questions and have someone monitor the chat to make sure all questions are addressed.
   - Be mindful of those dialing in by phone, as they might not have access to certain software features (e.g. hand raising).
 
+- **Meeting Recordings**
+  - Before recording a meeting let everyone know you are planning to record the meeting and only do so once you have consent from all participants.
+  - Let people know when the recording has started and when it has finished.
+
+- **Audio Transcriptions**
+  - Some people may want to use meeting transcription software to transcribe and summarize online meetings.
+  - Before activating the transcription software let everyone know you are planning to use the transcription software for the meeting and only do so once you have consent from all participants.
```

Comment:
Is it worth mentioning why this should have consent? I assume it's because all the data is being sent to a third-party service but it'd be nice to have something written down so people can make informed consent.